lib/puppet/environments.rb

```diff
@@ -268,8 +268,8 @@ def list
     # @!macro loader_get
     def get(name)
       @loaders.each do |loader|
-        if env = loader.get(name)
-          return env
+        if loader.get(name)
+          return loader.get(name)
```

Comment:
This code path gets called a lot, so we should keep the local `env` variable and not call `loader.get` twice.